src/libtorch.cc

```diff
@@ -1832,6 +1813,7 @@ ModelInstanceState::ReadOutputTensors(
 
     } else if (output_tensors[op_index].isList()) {
       // Custom handling for string/bytes tensor...
+      const int max_batch_size = model_state_->MaxBatchSize();
```

Comment:
You can store `support_batching` as member variable, then you don't need to compute the value again and again

Reply:
+1

Reply:
Done